gorush/notification_apns.go

```diff
@@ -358,7 +359,15 @@ func GetIOSNotification(req PushNotification) *apns2.Notification {
 
 	notification.Payload = payload
 
-	return notification
+	jsonMarshall, err := json.Marshal(notification)
```

Comment:
maybe add a flag to enable or disable showing the debug message.

Reply:
good idea, I will check it out